docs/source/usersguide/scripts.rst

```diff
@@ -0,0 +1,279 @@
+.. _usersguide_scripts:
+
+=======================
+Executables and Scripts
+=======================
+
+.. _scripts_openmc:
+
+----------
+``openmc``
+----------
+
+Once you have a model built (see :ref:`usersguide_basics`), you can either run
+the openmc executable directly from the directory containing your XML input
+files, or you can specify as a command-line argument the directory containing
+the XML input files. For example, if your XML input files are in the directory
+``/home/username/somemodel/``, one way to run the simulation would be:
+
+.. code-block:: sh
+
+    cd /home/username/somemodel
+    openmc
+
+Alternatively, you could run from any directory:
+
+.. code-block:: sh
+
+    openmc /home/username/somemodel
+
+Note that in the latter case, any output files will be placed in the present
+working directory which may be different from ``/home/username/somemodel``. If
+you're using the Python API, :func:`openmc.run` is equivalent to running
+``openmc`` from the command line. ``openmc`` accepts the following command line
+flags:
+
+-c, --volume           Run in stochastic volume calculation mode
+-g, --geometry-debug   Run in geometry debugging mode, where cell overlaps are
+                       checked for after each move of a particle
+-n, --particles N      Use *N* particles per generation or batch
+-p, --plot             Run in plotting mode
+-r, --restart file     Restart a previous run from a state point or a particle
+                       restart file
+-s, --threads N        Run with *N* OpenMP threads
+-t, --track            Write tracks for all particles
+-v, --version          Show version information
+-h, --help             Show help message
+
+.. _scripts_ace:
+
+----------------------
+``openmc-ace-to-hdf5``
+----------------------
+
+This script can be used to create HDF5 nuclear data libraries used by OpenMC if
+you have existing ACE files. There are four different ways you can specify ACE
+libraries that are to be converted:
+
+1. List each ACE library as a positional argument. This is very useful in
+   conjunction with the usual shell utilities (ls, find, etc.).
```

Comment:
Perhaps highlight `ls` and `find` as you do `convert` in the geometry visualization section.